tests/utils/host_utils.h

```diff
@@ -187,16 +187,12 @@ int fullLatticeIndex_5d_4dpc(int i, int oddBit);
 int process_command_line_option(int argc, char **argv, int *idx);
 int process_options(int argc, char **argv);
 
-#ifdef __cplusplus
-extern "C" {
-#endif
-
 // Implemented in face_gauge.cpp
-void exchange_cpu_sitelink(int *X, void **sitelink, void **ghost_sitelink, void **ghost_sitelink_diag,
+void exchange_cpu_sitelink(quda::lat_dim_t &X, void **sitelink, void **ghost_sitelink, void **ghost_sitelink_diag,
                            QudaPrecision gPrecision, QudaGaugeParam *param, int optflag);
-void exchange_cpu_sitelink_ex(int *X, int *R, void **sitelink, QudaGaugeFieldOrder cpu_order, QudaPrecision gPrecision,
+  void exchange_cpu_sitelink_ex(quda::lat_dim_t &X, quda::lat_dim_t &R, void **sitelink, QudaGaugeFieldOrder cpu_order, QudaPrecision gPrecision,
```

Comment:
whitespace? (or just do a bold clang-format of al files in the utils dir?)

Reply:
I'm happy to apply clang-format to all files in the utils dir.

Reply:
0e0282b240f7af80e9b410798e8a11075335d2c3